webapp/src/blocks.tsx

```diff
@@ -1010,7 +1016,8 @@ export class Editor extends toolboxeditor.ToolboxEditor {
                     errors={this.errors}
                     onSizeChange={this.onErrorListResize}
                     getErrorHelp={this.getErrorHelp}
-                    showLoginDialog={this.parent.showLoginDialog} />}
+                    showLoginDialog={this.parent.showLoginDialog}
+                    startDebugger={this.startDebugger} />}
```

Comment:
nit: move the `/>}` to a new line so if there's anything else that needs to be added to the ErrorList the old last line doesn't have to be affected.